test/ussd_registration.test.js

```diff
@@ -393,8 +474,9 @@ describe("Mama Nigeria App", function() {
                     .inputs(
                       {session_event: 'new'}  // dial in
                       , '12345'   // state_auth_code - personnel code
-                      , '6' // state_msg_receiver - friend_only
-                      , '09092222222'  // state_msisdn
+                      , '1' // state_msg_receiver - mother and father
+                      , '09092222222'  // state_msisdn_mother
+                      , '09091111111'  // state_msisdn_household
```

Comment:
i noticed a lot of 'friend_only'/'1' choices for state_msg_receiver so chose to change it and temporarily wanted to reach state_msg_language by a different branch of flow... i could change it back..?